web/yo/app/scripts/services/drugmaputils.js

```diff
@@ -11,9 +11,11 @@ angular.module('oncokbApp')
     .factory('drugMapUtils', function(OncoKB, _, $q, DatabaseConnector, $rootScope, ReviewResource, S, UUIDjs, $routeParams, firebaseConnector, firebasePathUtils) {
 
         function therapyStrToArr(key){
-            return key.split(",").map(function(element){
-                return element.trim().split(" + ")
-            });
+            if(key){
```

Comment:
This will return undefined for condition not passed, is this expected?

Reply:
Yes. Generally, it should always be passed, because the key is checked before calling this function. I set this condition just for testing. If not, split will lead to error.

Reply:
Should it return undefined or return empty string.